ENIGMAsystem/SHELL/Universal_System/var4.h

```diff
@@ -832,9 +832,9 @@ static inline bool is_ptr      (const variant &val) {
 namespace std {
 
 template<> class numeric_limits<var>: numeric_limits<double> {};
-template<> class numeric_limits<variant>: numeric_limits<double> {};
+// template<> class numeric_limits<variant>: numeric_limits<double> {};
 template<> class numeric_limits<const var&>: numeric_limits<double> {};
-template<> class numeric_limits<const variant&>: numeric_limits<double> {};
+// template<> class numeric_limits<const variant&>: numeric_limits<double> {};
```

Comment:
Is this still causing an issue? I think you can change this to `numeric_limits<enigma::variant>` or `numeric_limits<enigma_user::variant>` to prevent the lookup ambiguity.

Reply:
Great, it works perfectly with `numeric_limits<enigma_user::variant>`.